codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -53,6 +53,11 @@
 
         /**
          * Signal an error and release any retained buffers.
+         *<p>
+         * This method might be called multiple times.
```

Comment:
@buchgr Ah ok, so it's the encoder doing this, not the flow controller, correct?

Does it matter? Maybe the encoder's flow controlled objects should do nothing in `error()` after the first call?